recipe/meta.yaml

```diff
@@ -1,12 +1,12 @@
-{% set version = "0.25.0" %}
+{% set version = "dev_v1.26.0" %}
```

Comment:
this variable doesn't get used anywhere for dev PRs but it would be `0.26.0.dev1`